python/semantic_kernel/connectors/ai/mistral_ai/services/mistral_ai_text_embedding.py

```diff
@@ -33,7 +34,7 @@ def __init__(
         ai_model_id: str | None = None,
         api_key: str | None = None,
         service_id: str | None = None,
-        async_client: MistralAsyncClient | None = None,
+        async_client: Mistral | MistralAsyncClient | None = None,
```

Comment:
do we still need MistralAsyncClient? it is no longer needed and this is all breaking anyway?